src/components/Accordion/AccordionItem.tsx

```diff
@@ -13,10 +13,17 @@ export interface Props {
   // Make accordion item active or inactive.
   toggle?(index: number): void;
   style?:any;
+  clickHandler?(event: React.FormEvent<HTMLElement>) : void;
```

Comment:
@ankitsaraogi Can you double check and make sure the event name matches the conventions we're using in other components?  Like is it clickHandler or onClick or click?

Reply:
@harsimranb : I checked Table, also using clickHandler convention.